tests/unit/amo/reducers/test_userAbuseReports.js

```diff
@@ -0,0 +1,167 @@
+        let state = userAbuseReportsReducer(
+          initialState, showUserAbuseReportUI({ userId }));
+        state = userAbuseReportsReducer(state, showUserAbuseReportUI({ userId }));
+        state = userAbuseReportsReducer(state, sendUserAbuseReport({
+          errorHandlerId: 'some-error-handler',
+          message: 'foo',
+          userId,
+        }));
+        state = userAbuseReportsReducer(state, abortUserAbuseReport({ userId }));
+
+        expect(state).toMatchObject({
+          byUserId: {
+            [userId]: {
+              hasSubmitted: false,
+              isSubmitting: false,
+              uiVisible: false,
+            },
+          },
+        });
+      });
+    });
+
+    describe('hideUserAbuseReportUI', () => {
+      it('sets the uiVisible state to false', () => {
+        const userId = createUserAccountResponse();
+        const state = userAbuseReportsReducer(
+          initialState, hideUserAbuseReportUI({ userId }));
+
+        expect(state).toMatchObject({
+          byUserId: {
+            [userId]: { uiVisible: false },
+          },
+        });
+      });
+    });
+
+    describe('showUserAbuseReportUI', () => {
+      it('sets the uiVisible state to true', () => {
+        const userId = createUserAccountResponse().id;
+        const state = userAbuseReportsReducer(
+          initialState, showUserAbuseReportUI({ userId }));
+
+        expect(state).toMatchObject({
+          byUserId: {
+            [userId]: { uiVisible: true },
+          },
+        });
+      });
+    });
+
+    describe('sendUserAbuseReport', () => {
+      function defaultParams(params = {}) {
+        return {
+          errorHandlerId: 'some-error-handler',
+          message: 'The add-on is malware',
+          userId: createUserAccountResponse().id,
+          ...params,
+        };
+      }
+
+      it('dispatches sendUserAbuseReport', () => {
+        const params = defaultParams();
+        const action = sendUserAbuseReport(params);
+
+        expect(action.type).toEqual(SEND_USER_ABUSE_REPORT);
+        expect(action.payload).toEqual(params);
+      });
+    });
+
+    describe('loadUserAbuseReport', () => {
+      function abuseReportResponse(params = {}) {
+        return createFakeUserAbuseReport({
+          message: 'I am Groot!',
+          user: createUserAccountResponse({ id: 12 }),
+          ...params,
+        });
+      }
+
+      it('saves the abuse report response to the reducer', () => {
+        const abuseReport = abuseReportResponse();
+        const state = userAbuseReportsReducer(
+          initialState, loadUserAbuseReport({
+            message: abuseReport.message,
+            reporter: abuseReport.reporter,
+            userId: abuseReport.user.id,
+          }));
+
+        expect(state.byUserId[abuseReport.user.id].message)
+          .toEqual('I am Groot!');
+      });
+
+      it('requires a defined reporter', () => {
```

Comment:
Since you ported to `invariant`, you don't need this test. We can trust that `invariant()` works as advertised and I don't think it is helpful to add tests for static configuration.